packages/terra-form-input/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Added a dual means of disabling field component
```

Comment:
Can we update the changelog to be something about adding a new `disabled` prop?
```
### Added
* Added `disabled` prop to InputField component
```

Reply:
Will do!